examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>It doent average the sequencial input features? Otherwise, we couldnt use a BiLSTM.</p><p> </p><p>The text should be smth like:</p><p>"which concatenates the embedding vectors for all sequential features (item_id_list, f_47_list, f_68_list) per step (e.g. here 3)". The concatenated vectors are processed by a BiLSTM. The hidden state of the BiLSTM is concatenated with the embedding vectors of the categorical features"...</p><p> </p><p>"Then we connect it with a Multi-Layer Perceptron Block." -&gt; there is no sequential afterwards anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>I am a bit confused with your response Benedikt: <code>Otherwise, we couldnt use a BiLSTM.</code> I assume you have a simple MLP first, and then below you use BiLSTM.  why cant you concat the sequential lists with averaging if this is a simple MLP model?</p>